content/docs/user-guide/components/reference/paintbrush/paintbrush.md

```diff
@@ -0,0 +1,169 @@
+---
+linkTitle: Paint Brush
+title: Paint Brush
+description: ' The Paint Brush is a generalized tool for manipulating arbitrary data in world space using standard painting controls. '
+---
+
+In O3DE, many different components attach data to world space positions and regions. This data can be edited externally in content creation tools, but sometimes it can be easier to edit the data directly in the context of the assembled world inside of O3DE.
+
+The Paint Brush is an editing mode available on some types of components to enable standard painting controls for editing the component's data in world space.
+
+Some components directly map pixels from a source image into world space, but other components might have a more indirect mapping, such as weights stored on vertices on a mesh. The Paint Brush is designed to allow painting in world space for any arbitrary type of data. For that reason, the descriptions below refer more generically to "values" instead of "pixels", even though painting is generally considered to be an image-based operation. Also, the mapping of the data values into world space can be at different densities, so the descriptions below make a distinction between "world space" and "value space". "World space" represents a number of meters in the virtual world. "Value space" represents a number of pixels / vertex weights / etc.
```

Comment:
How does this sound for the last sentence? "Value space represents a quantity of adjacent data values in the data source being edited."

Why did you remove "considered to be"? I'm not _opposed_ to removing it, but I thought it provided additional meaning. It's like the difference between "I thought the kitten was ugly, even though the kitten was generally considered to be cute" and "I thought the kitten was ugly, even though the kitten was generally cute." Since the sentence is challenging an assumption or expectation, I thought the phrase "considered to be" provided meaning, not fluff.

Reply:
> How does this sound for the last sentence? "Value space represents a quantity of adjacent data values in the data source being edited."

Looks good!

Re: "Considered to be". I felt it was equivalent to generally; I agree there is a difference between the two.  "Considered to be" stresses the subjectivity of an evaluation, "generally" allows room for multiple evaluations while (subjectively?) preferring a default.  Keep your phrasing if you like it.